src/test/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorSupplierDiversityServiceImplTest.java

```diff
@@ -0,0 +1,159 @@
+package edu.cornell.kfs.pmw.batch.service.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.kuali.kfs.vnd.businessobject.VendorSupplierDiversity;
+import org.mockito.Mockito;
+
+import edu.cornell.kfs.pmw.batch.businessobject.KfsToPMWSupplierDiversityDTO;
+import edu.cornell.kfs.pmw.batch.businessobject.PaymentWorksVendor;
+import edu.cornell.kfs.pmw.batch.dataaccess.KfsSupplierDiversityDao;
+import edu.cornell.kfs.vnd.businessobject.CuVendorSupplierDiversityExtension;
+
+class PaymentWorksVendorSupplierDiversityServiceImplTest {
+    private static final String DAV_DESCRIPTION = "Disabled Veteran Owned Business";
+    private static final String DAV_NY_CODE = "NV";
+    private static final String EIGHTA_CODE = "8(A)";
```

Comment:
EIGHT_A_CODE might be easier to read than EIGHTA_CODE

Reply:
Ok